src/mgr/ActivePyModules.cc

```diff
@@ -929,10 +1001,36 @@ PyObject* ActivePyModules::get_latest_counter_python(
       f.dump_unsigned("v", datapoint.v);
     }
   };
-  return with_perf_counters(extract_latest_counters, svc_name, svc_id, path);
+  return with_unlabled_perf_counters(extract_latest_counters, svc_name, svc_id, path);
 }
 
-PyObject* ActivePyModules::get_perf_schema_python(
+PyObject *ActivePyModules::get_latest_counter_python(
+    const std::string &svc_name,
```

Comment:
Nit: please fix to maintain the formatting used for most of this file:
'PyObject* A' instead of 'PyObject *A'
and std::string& instead of std::string-blank&

Reply:
thank you! fixed it